DangoPlop/Assets/Scripts/Ball_Behavioiur.cs

```diff
@@ -50,18 +64,25 @@ public class Ball_Behavioiur : MonoBehaviour {
 		if (GameStart == true) {
 			Ball.transform.localScale = largeballscale;
 			GameStart = false;
-		} 
+		}
 
+        position = rb.transform.position;
 
-	}
+    }
 
     private void OnCollisionEnter2D(Collision2D coll)
     {
         if (coll.gameObject.tag == "Ball")
         {
             circle.isTrigger = true;
-
         }
+
+		if (coll.gameObject.tag == "Bottom") {
+			newSpeed.Set((float)rb.velocity.x,  Mathf.Sqrt ((float)(2 * 9.8 * (maxHeight - (-2.55)))));
```

Comment:
wHat's 2.55? If you don't have time to make the code to find it dynamically, we should just make the variable public for this so that we can change it to anythign we want.